src/main/resources/api/swagger-ui.html

```diff
@@ -48,14 +81,18 @@
           SwaggerUIStandalonePreset
         ],
         plugins: [
-          SwaggerUIBundle.plugins.DownloadUrl
+          SwaggerUIBundle.plugins.DownloadUrl,
+          cleanupPlugin
         ],
-        layout: "StandaloneLayout"
+        layout: 'StandaloneLayout',
+        defaultModelsExpandDepth: -1, // hide the huge list of schemas under the routes
+        defaultModelRendering: 'model', // schema has the descriptions, unlike the example value
+        defaultModelExpandDepth: 2, // affects the schema shown for a request or response
```

Comment:
I like these changes ... AND I would like to think about ways to make this reusable, so our swagger UIs are consistent across services once we have more MCTerra services. Same as previous comment: nothing to change in this PR, this is for future thinking.